### PR TITLE
feat(theme): add more social media icon & fix

### PR DESCRIPTION
**Pull request by CN-ycx**

### CODE REVIEW

File: packages/theme/templates/socialMediaIcons/twitch.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...e" calcMode="spline" dur="75ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/></path><path fill="#FFF" d="M512 345h42v125h-42V345zm146 0v125h-42V345h42z"><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="150ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="250ms" from="3 -3" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="0 0" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="50ms" from="3 -3" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="0 0" type="translate"/><animateTransform fill="freeze" accumulate="none" additive="replace" attributeName="transform" begin="indefinite" calcMode="spline" dur="75ms" from="0 0" keySplines="0.25 0.1 0.25 1" keyTimes="0; 1" restart="always" to="3 -3" type="translate"/></path></svg>
```
Comment (Mister-Hope) [inline packages/theme/templates/socialMediaIcons/twitch.svg:1]: Hi there, you should try to compress these icons, e.g.: remove `xml:space="preserve"` and `<animate>` tag:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" class="icon twitch-icon" viewBox="0 0 1024 1024"><circle cx="512" cy="512" r="512" fill="#9147FF"/><path fill="#FFF" d="M366 220 262 324v376h125v104l104-104h84l187-188V220z"/><path fill="#9147FF" d="m637 575 83-84V262H387v313h83v83l84-83z"/><path fill="#FFF" d="M512 345h42v125h-42V345zm146 0v125h-42V345h42z"/></svg>
```

**Review by Mister-Hope — COMMENTED** [review]
All icons should be added with a class `icon $name-icon`, e.g.: `icon baidu-icon`.

This is for style control consideration

**Review by Mister-Hope — CHANGES_REQUESTED** [review]
See comment above, related docs needs to be updated as well

**Comment by CN-ycx** [comment]
Some of icons from official websites, and that can't be compress, now trying redraw them. I has already used a similar compression tool like svgo to compress before submitting, and I'm still working on it

**Review by Mister-Hope — APPROVED** [review]
LGTM

**Comment by Mister-Hope** [comment]
@CN-ycx Could you update twitter icon in share component? https://plugin-components.vuejs.press/guide/utilities/share.html

**Comment by CN-ycx** [comment]
@Mister-Hope I noticed that in this component, some icons provide shape and icon version, and others only have icon or shape part.

Should I make shape and icon of these icons both?
Or we can import icons from social media icon like `../../../../../theme/templates/socialMediaIcons/*.svg`,
but the path is too long or illegal, maybe we can make icon as share component to use?

**Comment by Mister-Hope** [comment]
The component is expected to be as slim as possible, so if you can use a shape to generate a colorful icon, then no 2 svg.

**Comment by CN-ycx** [comment]
@Mister-Hope should all share icons added with a class `icon $name-icon`?

**Pull request merged**